renovate.json

```diff
@@ -14,5 +14,11 @@
   "suppressNotifications": [ "artifactErrors" ],
   "labels": [
     "dependency"
+  ],
+  "packageRules": [
+    {
+      "matchPackagePatterns": ["^registry.k8s.io/autoscaling/vpa-"],
+      "groupName": "dependency vertical pod autoscaler"
```

Comment:
We're using the `dependency ` prefix for the group name, so that the GitHub PR -> Jira integration picks up the dependency update PRs for the group.